### ISSUE TITLE
[Supporting the platform – Make Code - Control Pane] In high contrast (Black/White) mode 'Focus Indicator' does not visible.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3667]
User Experience:
Visual impaired user will find difficulty to see the control or element if that control does not adopt the high contrast mode, as they won't be able to take on that controls and they will miss the functionality of such control or element.

Note: User credentials should NOT be included in the bug.

Repro Steps:

1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Basic, Input, Music, Led, more, Radio" controls and select it.
6. Navigate through all the controls appearing after selection of element.
7. Verify the issue.

Actual Result:
In high contrast (Black/White) mode 'Focus Indicator' does not visible.

Expected Result:
In high contrast (Black/White) mode 'Focus Indicator' should be visible properly.

WCAG Reference:
[WCAG 4.3.1 - No Disruption of Accessibility Features](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B672c8a8b-ed4f-463f-868f-e8131caf9b74%7D%7D)

Test Environments:
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0)

**Comment by Shalini3457** [comment]
Labels:
#Accessibility; #A11ySev2; #HCL; #Benchmark-HCL-MakeCode-OCT2020; #WCAG4.3.1; #Win10-Edge(Chromium)

[Uploading WCAG4.3.1_High contrast focus.zip…]()

**Comment by shakao** [comment]
fixed with https://github.com/microsoft/pxt/pull/8063
